packages/@ourworldindata/grapher/src/color/ColorScale.ts

```diff
@@ -117,23 +119,21 @@ export class ColorScale {
         return this.manager.hasNoDataBin || false
     }
 
-    @computed get sortedNumericValues(): any[] {
-        return (
-            this.colorScaleColumn?.valuesAscending?.filter(
-                (x) => typeof x === "number"
-            ) ?? []
+    @computed get sortedNumericValues(): number[] {
+        return sortNumeric(
+            this.colorScaleColumn?.values.filter(R.isNumber) ?? []
         )
     }
```

Comment:
This change here actually fixes an issue we never knew we had, in https://ourworldindata.org/grapher/year-of-the-last-reported-rinderpest-case:
Jordan used to be colored in red (which means its color scale information is missing).

This was due to its main variable being a mixed string-and-number one, which meant that `valuesAscending` was sorted using string-compare and hence `sortedNumericValues[0]` is not the smallest value...